test/api/unit/models/build.test.js

```diff
@@ -132,6 +132,63 @@ describe('Build model', () => {
     });
   });
 
+  describe('build.url settting', () => {
+    let site;
+    before(async () => {
+      site = await factory.site({ defaultBranch: 'master', demoBranch: 'staging' });
+    });
+
+    it('default branch url start with site', async () => {
+      let build = await factory.build({ branch: site.defaultBranch, site });
+      build = await Build.findByPk(build.id);
```

Comment:
why do we need to requery for the build?

Reply:
initially url was assigned in aftercreate hook .. this is old junk i failed to remove